docs/sources/developers/kinds/_index.md

```diff
@@ -0,0 +1,10 @@
+---
+title: Grafana schema
+weight: 200
+_build:
+  list: false
```

Comment:
As discussed with @jdbaldry `draft: true` is not needed anymore so I removed it in this commit https://github.com/grafana/grafana/pull/60416/commits/967cab52fda006bed42e9fd59279a68f22c756a1